app/src/main/java/com/example/mymovieinfo/ui/theme/Shape.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.mymovieinfo.ui.theme
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Shapes
+import androidx.compose.ui.unit.dp
+
+val Shapes =Shapes(
+    small = RoundedCornerShape(4.dp),
+    medium = RoundedCornerShape(5.dp),
+    large = RoundedCornerShape(0.dp)
+//add more shapes
+)
```

Comment:
It looks like this group is using XML layouts.  If that's the case, Compose themes will not be used, and this can be discarded.